src/redux/store/appStore.js

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux'
+import todoAppReducer from '../reducers/appReducer'
+
+const store = createStore(todoAppReducer)
```

Comment:
Use here combine reducers. You have to think more perspective, will be easier to add new one if you use combine reducers.